### PR TITLE
Use `Dir.tmpdir` instead of explicit `/tmp`.

### PR DESCRIPTION
**Pull request by neonichu**

### CODE REVIEW

**Comment by neonichu** [comment]
cc @segiddins

File: lib/pod/command/try.rb [hunk 1]
```diff
@@ -63,7 +63,7 @@ def run
 
       # @return [Pathname]
       #
-      TRY_TMP_DIR = Pathname.new('/tmp/CocoaPods/Try')
+      TRY_TMP_DIR = Pathname.new(Dir.tmpdir) + Pathname.new('CocoaPods/Try')
```
Comment (segiddins) [inline lib/pod/command/try.rb]: Second part doesn't need to be wrapped in a pathname of its own

**Comment by segiddins** [comment]
Seems good

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
 # Cocoapods::Try Changelog
 
+## Master
+
+##### Bug Fixes
+
+* Use `Dir.tmpdir` instead of explicit `/tmp`.  
+  [Boris Bügling](https://github.com/)
```
Comment (kylef) [inline CHANGELOG.md]: `/neonichu`

Comment (neonichu) [inline CHANGELOG.md]: I AM GITHUB! 😜

Comment (neonichu) [inline CHANGELOG.md]: Fixed on master in 4b1b72e20188b0d96a7005aaecf7e332edf6ca3a

**Pull request merged**